src/components/Square.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
+  // For Wave 1 enable this
+  //  Component to alert a parent
   //  component when it's clicked on.
+  const clickHandler = (e) => {
+    e.preventDefault();
+    props.onClickCallback(props.id);
+  };
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return (
+    <button className="square" onClick={clickHandler}>
+      {props.value}
+    </button>
+  );
+};
```

Comment:
We could use object destructuring to reference prop data by their key name instead.

```
const Square = ( {value, id, onClickCallback, key} ) => {
  const clickHandler = (e) => {
    e.preventDefault();
    onClickCallback(id);
  };

  return (
    <button className="square" onClick={clickHandler}>
      {value}
    </button>
  );
```